contentctl/enrichments/cve_enrichment.py

```diff
@@ -18,7 +18,7 @@
 
 
 class CveEnrichmentObj(BaseModel):
-    id:Annotated[str, "^CVE-[1|2][0-9]{3}-[0-9]+$"]
+    id:Annotated[str, r"^CVE-[1|2]\d{3}-\d+$"]
```

Comment:
nit: PEP would have a space after the colon

```
id: Annotated[str, r"^CVE-[1|2]\d{3}-\d+$"]
```

Reply:
I will make this change, but for these types of formatting issues let's address these all-at-once in a giant PR that reformats all our Python files?
I am sure they are EVERYWHERE, but that doesn't mean we shouldn't address issues now, especially with code we've just touched.

Reply:
Agreed on an all-at-once PR! But I think we should also be in the habit of enforcing style guidelines during PR review; open to feedback on that point though